packages/storefront-sdk-plugins/commerce-queries/src/index.ts

```diff
@@ -62,17 +63,22 @@ export interface FetchCollectionEntriesMethodParams {
 	advancedOptions?: FetchMethodAdvancedParams;
 }
 
+export type CommerceQueriesResponse<QueryDocumentType> = Pick<
```

Comment:
So I think to match sdk v1, we might need to return the data value and not an object with the data property. ([sdk-v1 method return types](https://github.com/getnacelle/nacelle-storefront-sdk/blob/9d9aaf0e9be8cbba430742b22859c8b4a92f57f5/src/StorefrontClient/index.ts#L56-L93))

Reply:
Thank you @mdarrik for calling this out, I agree I removed the `{ data }` from the response. Added benefit is that it simplifies the code a good bit.